examples/app.R

```diff
@@ -238,6 +240,21 @@ css <- "
 #examples > div > .header {
   margin-top: 1em;
 }"
+format_name <- function(name, description) {
+  description <- ifelse(is.null(description), 'no data', description)
+  paste0('<strong>', name, '</strong> [', description, ']')
+}
+
+query_elastic_search <- function(query) {
+  resp_body = paste0('{"name_suggest": { "text": "',
+                     query,
+                     '", "completion": { "field": "name_suggest", "fuzzy": { "edit_distance": 1 } }} }')
+  response <- POST('http://temp_user:lfvbphk3635tcbbgv2@560e982eb806d23502d4ebb214386baa.eu-west-1.aws.found.io:9200/companies/_suggest?pretty',
```

Comment:
what is this example ElasticSearch server url + credentials? is this some official ElasticSearch demo? can you add a comment about where they come from?